applications/DEMApplication/custom_constitutive/DEM_continuum_constitutive_law.cpp

```diff
@@ -24,6 +24,40 @@ namespace Kratos {
     void DEMContinuumConstitutiveLaw::SetConstitutiveLawInProperties(Properties::Pointer pProp, bool verbose) const {
         if(verbose) KRATOS_INFO("DEM") << "Assigning DEMContinuumConstitutiveLaw to Properties " << pProp->Id() << std::endl;
         pProp->SetValue(DEM_CONTINUUM_CONSTITUTIVE_LAW_POINTER, this->Clone());
+        this->Check(pProp);
+    }
+
+    void DEMContinuumConstitutiveLaw::Check(Properties::Pointer pProp) const {
```

Comment:
When using continuum constitutive models, a discontinuum constitutive law is also always defined in the mdpa. Wouldn't it be redundant to define a check with the same variables being checked in the  DEMDiscontinuumConstitutiveLaw::Check  ?

Reply:
You are right that we are repeating code here, but currently, the continuum and discontinuum cl's are completely separate objects. I thought about linking them somehow, but we should discuss it carefully. Deriving one from the other is an option, but we might loose some flexibility when selecting them from the GUI. It is feasible, though. For the moment I treated them totally separately.